### PR TITLE
Docs: Describe how to configure Code formatter for IntelliJ IDEA

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: site/docs/community.md [hunk 1]
```diff
@@ -73,3 +73,12 @@ Iceberg has four mailing lists:
     - [Archive](https://lists.apache.org/list.html?issues@iceberg.apache.org)
 * **Private**: <private@iceberg.apache.org> -- private list for the PMC to discuss sensitive issues related to the health of the project
     - [Archive](https://lists.apache.org/list.html?private@iceberg.apache.org)
+
+
+## Setting up IDE and Code Style
+
+### Configuring Code Formatter for IntelliJ IDEA
+
+Navigate to **File > Settings > Editor > Code Style > Java**. Click on the gear wheel and select **Import Scheme** to import IntelliJ IDEA XML code style settings.
```
Comment (flyrain) [inline site/docs/community.md:82]: This differs from systems. It is "IntelliJ IDEA > Preferences > Editor > Code Style ..." in MacOS. We may just mention that "Go to Editor | Code Style".

File: site/docs/community.md [hunk 2]
```diff
@@ -73,3 +73,12 @@ Iceberg has four mailing lists:
...
+Point to [intellij-java-palantir-style.xml](../../.baseline/idea/intellij-java-palantir-style.xml) and hit **OK**. See also the [IntelliJ docs](https://www.jetbrains.com/help/idea/copying-code-style-settings.html)
+for additional details.
```
Comment (flyrain) [inline site/docs/community.md:84]: Can we at least mentioned that user needs to go to "Code > Reformat Code/File" to trigger it? Besides, I love the way that it formats the code automatically every time you save a file. I used it a lot in my Scala coding, but looks like we need a Macros to support it in IntelliJ, which is kind of cumbersome to be here.

**Comment by RussellSpitzer** [comment]
I didn't know you could do this, I just use the CheckstylePlugin

File: site/docs/community.md (hunk 2 above)
Comment (nastra) [reply site/docs/community.md]: done

File: site/docs/community.md (hunk 1 above)
Comment (nastra) [reply site/docs/community.md]: updated the description

**Comment by RussellSpitzer** [comment]
Thanks @nastra for writing and @flyrain for reviewing, this should be very helpful to new contributors!

**Comment by rdblue** [comment]
Thanks, @nastra!

**Pull request merged**